meta/main.yml

```diff
@@ -11,14 +11,21 @@ galaxy_info:
   author: Reto Gantenbein
   description: 'Setup Check_MK monitoring server'
   license: 'GPL-3.0'
-  min_ansible_version: '2.1.5'
+  min_ansible_version: '2.3.0'
```

Comment:
True, I didn't mention the reason here. Actually, I was bitten by ansible/ansible#14542. Ansible `2.3.0` fixed the issue for me. As mentioned in the linked PR it wasn't fixed in the `2.2.x` tree back then and I never tried to find a release who does. `2.3.0` is the safe choice. Will update the commit message accordingly.